public/views/children.html

```diff
@@ -46,7 +46,10 @@
                     <div class="col-sm-2 pre-active" style="text-align: center" >
                         <a style="text-align: center"  href="/views/children.html">Apadrina</a>
                     </div>
-                    <div class="col-sm-2 pre-active">
+                    <div class="col-sm-1" style="text-align: center" >
```

Comment:
actually, maybe if you change all three to col-md-1 the bar will all be on one line like it should be